src/modules/products/helpers.ts

```diff
@@ -13,4 +13,11 @@ export function getProductDHIS2Code(product: Product): string {
     const code = product.mappings.find(val => regex.test(val.systemName));
     if (code) return code.systemProductCode;
     return '';
+}
+
+export function getProductCode(product: Product, client: string): string|null {
```

Comment:
Does this, work in the following line of thought; if it's OpenLMIS or DHAMIS to this sort of thing and if it's any other client go by the default productCode as seen in OCL?
This should be also be asked about the Facilities caching and validation